Source/Shaders/Builtin/Functions/vertexLogDepth.glsl

```diff
@@ -7,6 +7,13 @@ void czm_updatePositionDepth() {
 #if defined(LOG_DEPTH) && !defined(DISABLE_GL_POSITION_LOG_DEPTH)
     v_logPositionEC = (czm_inverseProjection * gl_Position).xyz;
 
+#ifdef ENABLE_GL_POSITION_LOG_DEPTH_AT_HEIGHT
+    if (length(v_logPositionEC) < 2.0e6)
```

Comment:
The fix works... I'm just wondering if there is anything special about `2.0e6` or if it just happens to be a good value.

Reply:
I zoomed in to the edge of the polygon and slowly zoomed out until I saw the z fighting. Tested with larger and smaller polygons than what was in the bug example.

Reply:
Alright, thanks!